docs/release-notes/10.0.md

```diff
@@ -112,16 +112,16 @@ For a complete list of hardware support for which was added in this release, see
 
 There are three installation ISO images available:
 
-`AlmaLinux-10.0-x86_64-boot.iso` - a single network installation CD image that downloads packages over the Internet.
+`AlmaLinux-10.0-$arch-boot.iso` - a single network installation CD image that downloads packages over the Internet.
 
-`AlmaLinux-10.0-x86_64-minimal.iso` - a minimal self-containing DVD image that makes possible offline installation.
+`AlmaLinux-10.0-$arch-minimal.iso` - a minimal self-containing DVD image that makes possible offline installation.
 
-`AlmaLinux-10.0-x86_64-dvd.iso` - a full installation DVD image that contains mostly all AlmaLinux packages.
+`AlmaLinux-10.0-$arch-dvd.iso` - a full installation DVD image that contains mostly all AlmaLinux packages.
```

Comment:
unless I'm missing it, $arch isn't something we define anywhere in this doc, so we should define it here.